<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Login_Page">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <widget class="QPushButton" name="Login_Button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>100</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="User_Line">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="User_Label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
      <property name="alignment">
       <set>AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Password_Label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
      <property name="alignment">
       <set>AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password_Line">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Exchange_Page">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <widget class="QPushButton" name="Exit_Button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>300</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Convert_Button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>300</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Currency_1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Currency_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>From Currency</string>
      </property>
      <property name="alignment">
       <set>AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>To Currency</string>
      </property>
      <property name="alignment">
       <set>AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Converted</string>
      </property>
      <property name="alignment">
       <set>AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="Converted_Amount_Line">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Amount_Line">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
      <property name="alignment">
       <set>AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
